RoboFile.php

```diff
@@ -548,6 +548,20 @@ public function publish($opts = ['force' => false]) {
       ->run();
   }
 
+  function changelogUpdate($opts = ['version_name' => null]) {
+    $this->say("Updating changelog");
+    $outputs = $this->getChangelogController()->update($opts['version_name']);
+    if($opts['quiet']) {
+      return;
+    }
+    $this->say("Changelog \n{$outputs[0]} \n{$outputs[1]}\nRelease Notes \n" . $outputs[2] ?: 'none');
```

Comment:
"Release Notes" is a commonly used term for "Changelog". That's confusing. Let's rename this to something else maybe? Also - changelog has heading using "say", these notes output it as a string. Might be useful to unify these.

Reply:
What about "Internal Release Notes" or just "Internal notes" ?

Reply:
Yeah, I don't have a good naming but this could even scream a bit... like "IMPORTANT NOTES" or something like that. I guess it usually is quite important indeed.